File: common/src/main/java/com/xpf/common/cons/ApiRequestUrl.java
package com.xpf.common.cons;

import com.xpf.common.BuildConfig;

/**
 * Created by xpf on 2016/11/12 :)
 * Wechat:18091383534
 * Function:提供当前应用访问服务器的请求地址
 */
public class ApiRequestUrl {

    public static final String HOST = BuildConfig.HOST; // 本地服务器地址

    public static final String BASE_URL = HOST + "/iMoney/";

    public static final String INDEX = BASE_URL + "index"; // 访问首页数据

    public static final String LOGIN = BASE_URL + "login"; // 访问登录的url

    public static final String PRODUCT = BASE_URL + "product"; // 访问“所有理财”的url

    public static final String UPDATE = BASE_URL + "update"; // 访问服务器端当前应用的版本信息

    public static final String STUDENTS = BASE_URL + "students";
}

File: app/src/main/java/com/xpf/p2p/constants/ApiRequestUrl.kt
package com.xpf.p2p.constants

import com.xpf.p2p.BuildConfig

/**
 * Created by xpf on 2016/11/12 :)
 * Function:提供当前应用访问服务器的请求地址
 */
object ApiRequestUrl {
    const val HOST: String = BuildConfig.HOST
    const val BASE_URL: String = "$HOST/iMoney/"
    const val INDEX: String = "${BASE_URL}index"
    const val LOGIN: String = "${BASE_URL}login"
    const val PRODUCT: String = "${BASE_URL}product"
    const val UPDATE: String = "${BASE_URL}update"
    const val STUDENTS: String = "${BASE_URL}students"
}
